Block Sync: Syncing latest blocks using IPLD or Shrex (#200)
celestiaorg/test-infra: docs/test-plans/005-Block-Sync/64-128sq-plan.md

| Block Square Size<br />`X` |            Bridge Nodes amount <br />`Y`             |  Full Nodes Amount <br />`Z` | Peer Limit <br />`W` |
| :---------------------------: | :-----------------------------------------: | :---------------------------: | :---------------------------: |
|              64               | 3 | 32 | 3 |  
|              128               |  3  | 32 | 3 | 

And For Historical Scenarios

| Block Square Size<br />`X` |            Bridge Nodes amount <br />`Y`             |  Full Nodes Amount <br />`Z` | Historical Full Nodes <br />`H` | Peer Limit <br />`W` |
| :---------------------------: | :-----------------------------------------: | :---------------------------: | :---------------------------: | :---------------------------: |
|              64               | 3 | 12 | 12 |   3 |  
|              128               |  3  | 12 | 12 |  3 |

## HW Resources setup for Block Square Size ==  64|128|256

We already know that we can count on 172 cores CPU and 700 GB of RAM to allocate for testable nodes
Hence, our network setup/HW allocation will look like this:

1. 1 Validators with each having 1 CPU and 4 Gb
2. 3 Bridge Nodes with each having 1 CPU and 8 Gb

> 3. 32 Full Nodes with each having 1 CPU and 8 Gb

Reviewer: I guess its 4-4-4 CPUs instead of 1-1-1?

## Network Bandwidth:

- Validator/Bridge/Full: 256Mib -- 50ms

## Network Topology

### All cases

> For all cases, the singular validator will be connected to all three bridges.

Reviewer: ```suggestion
For all cases, the singular validator is used for three bridges.
```

### Case 1 - Syncing Latest:

> Z Full nodes all connected to some or all of the existing Y bridges, and connected amongst themselves (_peerLimit is set to W_) such that full nodes are syncing latest blocks as they are being produced.

Reviewer: ```suggestion
Z Full nodes all connected to all of the existing Y bridges, and connected amongst themselves (_peerLimit is set to W_) such that full nodes are syncing latest blocks as they are being produced.
```

Reviewer: Or please be more precise where the `some` is defined in test data

Author: some means exactly one, as per the logic that picks bridge nodes for fulls or lights, correcting...

> This will include three main networking variations for each getter:

Reviewer: ```suggestion
This will include three main networking variations for each Full nodes' getters:
```
